content/en/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -146,7 +146,7 @@ <h3>View the app</h3>
                 <p><b><code>export POD_NAME=$(kubectl get pods -o go-template --template '{{range .items}}{{.metadata.name}}{{"\n"}}{{end}}')</code></b><br />
                    <b><code>echo Name of the Pod: $POD_NAME</code></b></p>
                 <p>You can access the Pod through the proxied API, by running:</p>
-                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/</code></b></p>
+                <p><b><code>curl http://localhost:8001/api/v1/namespaces/default/pods/$POD_NAME/proxy/</code></b></p>
```

Comment:
I believe localhosts:8001 is already a proxy, why are we appending a suffix?

Reply:
Ah I see what you mean. I had mis-understood that "accessing the pod" would mean accessing the http interface of the pod itself. Rather than the http interface of the pod api.